### PR TITLE
hledger-balance: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/hledger-balance.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# hledger balance
+
+> Un informe «sumatorio» flexible y de propósito general que muestra cuentas con algún tipo de dato numérico.
```
Comment (spageektti) [inline pages.es/common/hledger-balance.md:3]:
```suggestion
> Un informe "sumatorio" flexible y de propósito general que muestra cuentas con algún tipo de dato numérico.
```

**Review by tricantivu — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Review by kant — COMMENTED** [review]
LGTM

**Comment by simonmichael** [comment]
Thanks for translating!

**Comment by simonmichael** [comment]
@kant is there any chance you'd have time to translate the other hledger pages ? There are 11 in total.

**Pull request merged**